### DISCUSSION TITLE
Procesondersteuning / inzet proces enging

### DISCUSSION THREAD

**Discussion by sytskevanhasselt** [Klantinteractie-Servicesysteem/KISS-frontend#33] [General]
Het proces voor terugbelverzoeken binnen KISS is zodanig licht dat de inzet van Camunda / Valtimo overkill is. We regelen dit proces codematig in. 

Terugbelverzoek wordt NIET in release 1 opgepakt. We kunnen deze discussie als nodig ook nog tijdens release 1 voeren.

Dit ligt in het verlengde van de discussie waar maken we een zaak van.

**Comment by rubenvdlinde** [comment]
te bespreken op architectuur overleg van 2 Juni 12:30

Wie hebben we nodig
- [ ] Architecten
